build/source/engine/coupled_em.f90

```diff
@@ -720,9 +765,9 @@ lightSnow            ! maximum interception capacity an inverse function of new
                     mpar_data%var(iLookPARAM%tempScalOvrbdn),               & ! intent(in): temperature scaling factor for overburden pressure (K-1)
                     mpar_data%var(iLookPARAM%baseViscosity),                 & ! intent(in): viscosity coefficient at T=T_frz and snow density=0 (kg m-2 s)
                     ! intent(inout): state variables
-                    mvar_data%var(iLookMVAR%mLayerDepth)%dat(1:nSnow),      & ! intent(inout): depth of each layer (m)
-                    mvar_data%var(iLookMVAR%mLayerVolFracLiq)%dat(1:nSnow), & ! intent(inout):  volumetric fraction of liquid water after itertations (-)
-                    mvar_data%var(iLookMVAR%mLayerVolFracIce)%dat(1:nSnow), & ! intent(inout):  volumetric fraction of ice after itertations (-)
+                    prog_data%var(iLookPROG%mLayerDepth)%dat(1:nSnow),      & ! intent(inout): depth of each layer (m)
+                    prog_data%var(iLookPROG%mLayerVolFracLiq)%dat(1:nSnow), & ! intent(inout):  volumetric fraction of liquid water after itertations (-)
+                    prog_data%var(iLookPROG%mLayerVolFracIce)%dat(1:nSnow), & ! intent(inout):  volumetric fraction of ice after itertations (-)
                     ! output: error control
                     err,cmessage)                     ! intent(out): error control
     if(err/=0)then; err=55; message=trim(message)//trim(cmessage); return; endif
```

Comment:
This would be a good one to add. My convention is as follows:
0 = OK
positive = fatal error
negative = warning (or recoverable error)

We should really use named variables. I'll open up a new PR.